Library/Homebrew/test/testing_env.rb

```diff
@@ -1,6 +1,18 @@
 $:.unshift File.expand_path("../..", __FILE__)
 $:.unshift File.expand_path("../lib", __FILE__)
 
+# This must be at the top
+if ENV["HOMEBREW_TESTS_COVERAGE"]
+  require "simplecov"
+  SimpleCov.start do
+    SimpleCov.root File.expand_path("#{File.dirname(__FILE__)}/..")
+    add_filter do |src|
+      src.filename.include?("Homebrew/test") ||
+      src.filename.include?("vendor/bundle")
+    end
+  end
```

Comment:
Might be good to add an aspirational `# minimum_coverage 50` (or something lower or equal to the current value) to gently stop us from adding new code without tests (I'm guilty).